### PR TITLE
Allow commanders and admins to edit game options.

### PR DESCRIPTION
**Pull request by CalebSerafin**
## What type of PR is this.
* Bug
* Change
* [x] Enhancement

### What have you changed?
* All button changes to Civ Limit, AI Limit, Spawn Distance create a log entry.

### Why?
* Allow commanders and admins to edit game options.
* Slow min and max to user.
* Put blame on server people who raise it. (Allow them to be educated).
* Know its effect on performance.

### Please specify which Issue this PR Resolves.
closes https://github.com/official-antistasi-community/A3-Antistasi/issues/1772

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
* Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
1. Load Antistasi
2. Click each of the limit buttons at the Map Object > Game Options.
3. Check RPT.

********************************************************
Notes:
```sqf
191.156: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed Spawn Distance from 1000 to 900
194.828: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed Civilian Limit from 5 to 6
195.859: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed Civilian Limit from 6 to 7
196.047: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed Civilian Limit from 7 to 8
200.953: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed AI Limit from 150 to 140
201.765: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed AI Limit from 140 to 130
202.281: [Antistasi] | INFO | fn_HQSpawnOptions.sqf | SET | FrostsBite [12345678901234567] [4] changed AI Limit from 130 to 120
```
```sqf
487.765: [Antistasi] | ERROR | fn_HQSpawnOptions.sqf | ACCESS VIOLATION | FrostsBite [12345678901234567] [4] attempted calling restricted backing method [R Alpha 1-2:1 (FrostsBite) REMOTE,"maxUnits","increase"]
```
```sqf
694.187: [Antistasi] | ERROR | fn_HQSpawnOptions.sqf | HACKING | Petros [] [2] attempted impersonating FrostsBite [12345678901234567] [4] while calling [R Alpha 1-2:1 (FrostsBite) REMOTE,"maxUnits","increase"]
```
Game Options Stats
Game Options Slider

### CODE REVIEW

**Comment by CalebSerafin** [comment]
Force push changes

**Comment by CalebSerafin** [comment]
Note: The last PR's feature set was not complete yet, this one is.

File: A3-Antistasi/functions/init/fn_initClient.sqf [hunk 1]
```diff
@@ -455,7 +455,20 @@ fireX allowDamage false;
...
+	[
+		"Game Options",
+		"Version: "+ antistasiVersion +
+		"<br/><br/>Difficulty: "+ ( ["Easy","Normal","Hard"] select ((skillMult-1) min 2) ) +
+		"<br/>Unlock Weapon Number: "+ str minWeaps +
+		"<br/>Limited Fast Travel: "+ (["No","Yes"] select limitedFT) +
+		"<br/>AI Limit: "+ str maxUnits +
+		"<br/>Spawn Distance: "+ str distanceSPWN + "m" +
+		"<br/>Civilian Limit: "+ str civPerc
+	] call A3A_fnc_customHint;
```
Comment (HakonRydland) [inline A3-Antistasi/functions/init/fn_initClient.sqf:468]: everything but the creation of the dialgo should imo be handled by the dialog onLoad event, but thats can wait for UI rework.

**Review by HakonRydland — APPROVED** [review]
i can't see any errors in here, the slider was fancy 👍

**Pull request merged**